hawkular-alerts-actions/hawkular-alerts-actions-plugins/hawkular-alerts-actions-email/hawkular-alerts-actions-email-plugin/src/main/java/org/hawkular/alerts/actions/email/EmailPlugin.java

```diff
@@ -289,11 +289,11 @@ protected Message createMimeMessage(ActionMessage msg) throws Exception {
         Status status = alert != null && alert.getStatus() != null ? alert.getStatus() : Status.OPEN;
         String statusStr = status.name().toLowerCase();
 
-        String from = props.get(PROP_FROM + "." + statusStr);
+        String from = props.get(PROP_FROM + "" + statusStr);
```

Comment:
oh, I'm sure I corrected that, I guess this was introduced by the IDE for some reason when moving files.
Thanks, this is a good catch I will review it.